k-distribution/tests/regression-new/kprove-branchingAllowed/Makefile

```diff
@@ -3,5 +3,6 @@ EXT=def
 KOMPILE_BACKEND=haskell
 TESTDIR=.
 KPROVE_FLAGS=--branching-allowed 1 --spec-module DEF-SPEC
+export KORE_EXEC_OPTS=--log-level error
```

Comment:
So here are we basically just telling it to be quiet about warnings for this test?

Reply:
Yes, we used this before in several other tests.
I like tests that are focused on a specific problem.
Since the haskell-backend can give warnings with location information and all sorts of runtime information this option is very useful.